packages/template-retail-react-app/app/utils/utils.js

```diff
@@ -217,3 +217,14 @@ export const buildRedirectURI = (appOrigin = '', redirectPath = '') => {
         return ''
     }
 }
+
+/**
+ * Retrieves the value of a specified cookie.
+ * @param {string} name - name of cookie to retrieve
+ * @returns {string|null} cookie value if found, otherwise `null`
+ */
+export const getCookie = (name) => {
```

Comment:
The [template](https://github.com/SalesforceCommerceCloud/pwa-kit/blob/develop/packages/template-retail-react-app/package.json#L70) has installed `js-cookie` which should allow you to get the cookie with `Cookies.get('sid')` without using this util. https://github.com/js-cookie/js-cookie